blueprints/data-solutions/vertex-mlops/README.md

```diff
@@ -0,0 +1,93 @@
+
+
+##  GCP resources
+A terraform script is provided to setup all the required resources:
+
+- GCP Project  to host all the resources
+- Isolated VPC network and a subnet to be used by Vertex and Dataflow (using a Shared VPC is also possible). 
+- Firewall rule to allow the internal subnet communication required by Dataflow
+- Cloud NAT required to reach the internet from the different computing resources (Vertex and Dataflow)
+- GCS buckets to host Vertex AI and Cloud Build Artifacts. By default the buckets will be regional and should match the Vertex AI region for the different resources (i.e. Vertex Managed Dataset) and processes (i.e. Vertex trainining)
+- BigQuery Dataset where the training data will be stored. This is optional, since the training data could be already hosted in an existing BigQuery dataset.
+- Service account `mlops-[env]@` with the minimum permissions required by Vertex and Dataflow
+- Service account `github` to be used by Workload Identity Federation, to federate Github identity.
+- Secret to store the Github SSH key to get access the CICD code repo.
+
+![MLOps project description](./images/mlops_projects.png "MLOps project description")
+
+## Pre-requirements
+
+### User groups
+
+User groups provide a stable frame of reference that allows decoupling the final set of permissions from the stage where entities and resources are created, and their IAM bindings defined. These groups should be created before launching Terraform.
+
+We use the following groups to control access to resources:
+
+- *Data Scientits* (gcp-ml-ds@<company.org>). They create ML pipelines in the experimentation environment.
+- *ML Engineers* (gcp-ml-eng@<company.org>). They handle and run the different environments, with access to all resources in order to troubleshoot possible issues with pipelines. 
+
+These groups are not suitable for production grade environments. You can configure the group names through the `groups`variable. 
+
+### Git environment for the ML Pipelines
+
+Make sure you have ready a Github repo with the ML pipeline code. 
+You can clone the following example for setting up the repo: https://github.com/pbalm/professional-services/tree/vertex-mlops/examples/vertex_mlops_enterprise
+This repo should have at least one of the following branches: `dev`, `staging`, `prod`
+
+You will need to configure the Github organization and repo name in the `identity_pool_claims` variable.
+
+##  Instructions
+###  Deploy the experimentation environment
+
+- Create a `terraform.tfvars` file and specify the required variables. You can use the `terraform.tfvars.sample` an an starting point
+
+```tfm
+project_create = {
+    billing_account_id = "000000-123456-123456"
+    parent             = "folders/111111111111"
+}
+project_id          = "creditcards-dev"
+```
+- Make sure you fill in the following parameters:
```

Comment:
we have variable description already below, I would remove from here.

Reply:
OK